src/rosdoc_lite/rdcore.py

```diff
@@ -150,6 +150,12 @@ def instantiate_template(tmpl, tempvars):
     """
     looks up file within rosdoc_lite package, return its content, may sys.exit on error
     """
-    for k, v in tempvars.iteritems():
+    try:
+        items = tempvars.items()
```

Comment:
I believe that we should be able to just use `items()` in both 2.7 and 3.x

http://wiki.ros.org/python_2_and_3_compatible_code